recipe/meta.yaml

```diff
@@ -54,26 +60,37 @@ source:
   {% endif %}
 
 build:
-  number: 3
-  skip: True  # [blas_impl == 'openblas' and win]
+  number: 4
+  # Note: 1.16 was never built for osx-arm64
+  skip: True  # [(blas_impl == 'openblas' and win) or py<36 or py>39 or (osx and arm64)]
   force_use_keys:
     - python
 
+requirements:
+  build:
+    - patch  # [not win]
+    - m2-patch  # [win]
+
 outputs:
   # this one has all the actual contents
   - name: numpy-base
     script: install_base.sh   # [unix]
     script: install_base.bat  # [win]
+    build:
+      ignore_run_exports:
+        - libgfortran5  # [osx]
+      missing_dso_whitelist:
+        - '$RPATH/ld64.so.1'  # [s390x]
```

Comment:
I think we can add entry_points https://github.com/numpy/numpy/blob/3dec7099ce38cb189880f6f69df318f35ff9a5ea/setup.py#L385

Reply:
Since this package is built directly from its setup.py, the scripts are also installed in the process, you can see that in the tarball of an older build:

```
numpy-1.16/pkgs$ ls
numpy-base-1.16.6-py39hdc34a94_3.tar.bz2

...unpack...

numpy-1.16/pkgs$ ls bin/
f2py  f2py3  f2py3.9
```